src/Compilers/CSharp/Portable/Symbols/TupleTypeSymbol.cs

```diff
@@ -303,6 +308,158 @@ public override bool IsSealed
             }
         }
 
+        public override int Arity
+        {
+            get
+            {
+                return _underlyingType.Arity;
+            }
+        }
+
+        public override ImmutableArray<TypeParameterSymbol> TypeParameters
+        {
+            get
+            {
+                return _underlyingType.TypeParameters;
+            }
+        }
+
+        internal override ImmutableArray<ImmutableArray<CustomModifier>> TypeArgumentsCustomModifiers
+        {
+            get
+            {
+                return _underlyingType.TypeArgumentsCustomModifiers;
+            }
+        }
+
+        internal override bool HasTypeArgumentsCustomModifiers
+        {
+            get
+            {
+                return _underlyingType.HasTypeArgumentsCustomModifiers;
+            }
+        }
+
+        internal override ImmutableArray<TypeSymbol> TypeArgumentsNoUseSiteDiagnostics
+        {
+            get
+            {
+                return _underlyingType.TypeArgumentsNoUseSiteDiagnostics;
+            }
+        }
+
+        public override NamedTypeSymbol ConstructedFrom
+        {
+            get
+            {
+                return _underlyingType.ConstructedFrom;
+            }
+        }
+
+        public override bool MightContainExtensionMethods
+        {
+            get
+            {
+                return false;
+            }
+        }
+
+        public override string Name
+        {
+            get
+            {
+                return string.Empty;
+            }
+        }
+
+        internal override bool MangleName
+        {
+            get
+            {
+                return _underlyingType.MangleName;
+            }
+        }
+
+        public override IEnumerable<string> MemberNames
+        {
+            get
+            {
+                return _fields.Select(f => f.Name);
+            }
+        }
+
+        internal override bool HasSpecialName
+        {
+            get
+            {
+                return _underlyingType.HasSpecialName;
+            }
+        }
+
+        internal override bool IsComImport
+        {
+            get
+            {
+                return _underlyingType.IsComImport;
+            }
+        }
+
+        internal override bool IsWindowsRuntimeImport
+        {
+            get
+            {
+                return _underlyingType.IsWindowsRuntimeImport;
+            }
+        }
+
+        internal override bool ShouldAddWinRTMembers
+        {
+            get
+            {
+                return _underlyingType.ShouldAddWinRTMembers;
+            }
+        }
+
+        internal override bool IsSerializable
+        {
+            get
+            {
+                return _underlyingType.IsSerializable;
```

Comment:
underlying type in current implementation is actually serializable. I think this bit for the most part affects the run time and tuple will not exist at run time. It still feels like deferring to underlying is a better option.